### PR TITLE
Automation for CSPL-1089

### PR DESCRIPTION
**Pull request by pdhanoyasplunk**
Added/updated automation scenario for MC CRD
- Added MC with M4 SVA ( New Case)
- Modified existing case to work with MC CRD for C3

### CODE REVIEW

File: test/monitoring_console/monitoring_console_test.go [hunk 1]
```diff
@@ -396,12 +411,22 @@ var _ = Describe("Monitoring Console test", func() {
 			// Ensure search head cluster go to Ready phase
 			testenv.SearchHeadClusterReady(deployment, testenvInstance)
 
-			// Wait for Monitoring Console Pod to be in READY status
-			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+			// Deploy Monitoring Console CRD
```
Comment (kashok-splunk) [inline test/monitoring_console/monitoring_console_test.go:414]: Nit: Deploy Monitoring Console Pod

File: test/monitoring_console/monitoring_console_test.go [hunk 2]
```diff
@@ -452,9 +477,19 @@ var _ = Describe("Monitoring Console test", func() {
 			// Ensure Indxer cluster scales up and go to ScalingUp phase
 			testenv.VerifyIndexerClusterPhase(deployment, testenvInstance, splcommon.PhaseScalingUp, idxcName)
 
-			// Deploy Standalone
-			standalone, err := deployment.DeployStandalone(deployment.GetName())
-			Expect(err).To(Succeed(), "Unable to deploy standalone instance ")
+			spec := enterpriseApi.StandaloneSpec{
```
Comment (kashok-splunk) [inline test/monitoring_console/monitoring_console_test.go:480]: Nit: need comment here

File: test/monitoring_console/monitoring_console_test.go [hunk 3]
```diff
@@ -492,4 +535,67 @@ var _ = Describe("Monitoring Console test", func() {
...
+			// Ensure indexers go to Ready phase
+			testenv.IndexersReady(deployment, testenvInstance, siteCount)
+
+			// Ensure indexer clustered is configured as multisite
+			testenv.IndexerClusterMultisiteStatus(deployment, testenvInstance, siteCount)
+
+			// Ensure search head cluster go to Ready phase
+			testenv.SearchHeadClusterReady(deployment, testenvInstance)
+
+			// Deploy Monitoring Console CRD
```
Comment (kashok-splunk) [inline test/monitoring_console/monitoring_console_test.go:570]: super nit: Deploy Monitoring Console Pod

File: test/monitoring_console/monitoring_console_test.go [hunk 4]
```diff
@@ -492,4 +535,67 @@ var _ = Describe("Monitoring Console test", func() {
...
+			Expect(err).To(Succeed(), "Unable to deploy Monitoring Console instance")
+
+			// Verify Monitoring Console is Ready and stays in ready state
+			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
+
+			// Get contents of MC config map
+			mcConfigMap, err := testenv.GetMCConfigMap(deployment, testenvInstance.GetName(), mcName)
+			Expect(err).To(Succeed(), "Unable to get MC config map")
+
+			// Check Monitoring console is configured with all search head instances in namespace
```
Comment (kashok-splunk) [inline test/monitoring_console/monitoring_console_test.go:581]: Should we also check if cluster master is present in configured peers list?

Comment (pdhanoyasplunk) [reply test/monitoring_console/monitoring_console_test.go:581]: The CM part if checked indirectly when we verify Indexers

**Review by kashok-splunk — APPROVED** [review]

**Pull request merged**